leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionServiceBuilder.java

```diff
@@ -99,6 +105,7 @@ public PaxosLeaderElectionService build() {
                 executor,
                 pingRateMs,
                 randomWaitBeforeProposingLeadershipMs,
+                noQuorumMaxDelayMs,
```

Comment:
OK. Could we do a preconditions-check that this is non-negative? Otherwise if someone gives a negative value we'll only fail when we try to generate the range / in current impl, when we sleep for negative time.